airflow/www_rbac/views.py

```diff
@@ -2567,6 +2600,32 @@ def action_set_success(self, drs, session=None):
             flash('Failed to set state', 'error')
         return redirect(self.get_default_url())
 
+    @action('set_canceled', "Set state to 'canceled'",
+            "All task instances for the selected DAG Runs will be canceled. Are you sure?",
+            single=False)
+    @provide_session
+    def action_set_canceled(self, drs, session=None):
+        try:
+            DR = models.DagRun
+            dagrun_count = 0
+            task_instance_count = 0
+            dirty_ids = []
+            for dr in session.query(DR).filter(
+                DR.id.in_([dagrun.id for dagrun in drs])).all():
+                dirty_ids.append(dr.dag_id)
```

Comment:
are we using this variable for anything?

Reply:
Nope. Deleted. Copied this from the other endpoint.